README.md

```diff
@@ -1,16 +1,22 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
 - Recursion
+Recursion : See recursion
 - Recursive Case
+A problem that can be divided into smaller problems
```

Comment:
Hmm... this is true. But not the definition of recursion. Recursion is specifically when a method calls itself, directly or indirectly.